app/components/sort_component.rb

```diff
@@ -0,0 +1,42 @@
+# When there are only two sorting options, these should be presented as links, not
+# options in a drop-down. This component manages that display logic.
+class SortComponent < GovukComponent::Base
+  attr_reader :url_params, :path, :sort
+
+  # @param [Method] path The helper method which generates the destination path for the links or forms
```

Comment:
Another nitpick: This argument takes a `String` (we just happen to pass in a method that returns a string)

Reply:
Actually, I'm passing a method itself (e.g. `method(:page_path)`) because I need to be able to pass the `url_params` to that method to do with as it sees fit (merged with the sort_by param).